bridge/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+// For more info on versioning, see the README.
+val version = "0.1.0"
+
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.kotlin.parcelize)
+}
+
+android {
+    namespace = "com.bitwarden.bridge"
+    compileSdk = 34
+
+    defaultConfig {
+        minSdk = 28
```

Comment:
Do we want to match the minSDdk from the app?

Reply:
This matches the minSdk of the authenticator app 👍

Reply:
Should we leave a comment about this maybe?

```
// This min value is selected to accommodate known consumers
```

Something like that? Or we could even go lower if necessary too.

Reply:
I like that:

732653f740bba5ef6a6a14a51ba3351166337f03